environment_scenes/World.gd

```diff
@@ -1,5 +1,5 @@
 extends "res://Base.gd"
-
```

Comment:
Could you undo this file? I'll make the client/server interaction make the maps to the next one